assets/js/modules/analytics/datastore/settings.test.js

```diff
@@ -166,32 +154,30 @@ describe( 'modules/analytics settings', () => {
 					...fixtures.propertiesProfiles.profiles[ 0 ],
 					id: '987654321',
 				};
-
-				fetch
-					.doMockOnceIf(
-						/^\/google-site-kit\/v1\/modules\/analytics\/data\/create-profile/
-					)
-					.mockResponseOnce(
-						JSON.stringify( createdProfile ),
-						{ status: 200 }
-					)
-					.doMockOnceIf( /^\/google-site-kit\/v1\/modules\/analytics\/data\/settings/ )
-					.mockResponseOnce( async ( req ) => {
-						const { data } = await req.json();
-						// Return the same settings passed to the API.
```

Comment:
Please keep this comment

Reply:
:+1:

Reply:
Got it. Returned this comment to its rightful place.